### ISSUE TITLE
Wrong timezone on csv export

### ISSUE THREAD

**Issue by joortwein** [ioBroker/ioBroker.history#271]
when i export data to csv, the timestamps are shown 2 hours lower than the ones in the gui.
for example the timestamp in the gui is "2023-08-29 20:00:00" but after export it es shown as "2023-08-29 18:00:00"

German timezone is my default. Seems the export routine converts to UTC.
This is very confusing during troubleshooting my datarows

History Adapter is v2.2.6
Data is stored in influxdb 1.8

**Comment by Apollon77** [comment]
Whats the timezone of yout hostsystem? assumption: it is UTC and not "german summer time" ... fix there and files are fixed too

**Comment by joortwein** [comment]
No, timezone on the server is CEST and shows exactly the same time as the client.

Influx has the behavior, when changing the time format with "precision rfc3339", all listed dates are shown in UTC.
To display the right time (+2) there must be a command at the end of the query in the form like "tz('CET')"

In a query it looks like the following
select * from "javascript.0.TasmotaVerbrauch.StromZaehler.aktueller_wert" where time <=1693342850000000000 order by desc limit 10 tz('CET')

Possibly the csv function at the history adapter doesn't consider this behavior?

**Comment by Apollon77** [comment]
Also our influxdb adapter do not have this addition btw

**Comment by joortwein** [comment]
What I doesn't understand is, that the time in the history gui is ok, but not in the csv data.
Both queries should work over the influxdb adapter,or?

**Comment by Apollon77** [comment]
That CSV is generated by Admin ... normally the data coming from server are"just" numbers .. so it more seems to be an iadmin issue when? @foxriver76 could you please have a look in admin when it comes to this CSV generation?
